holidays/countries/sao_tome_and_principe.py

```
@@ -0,0 +1,121 @@
+        "P",
+    )
+    subdivisions_aliases = {
+        # Districts.
+        "Água Grande": "01",
+        "Cantagalo": "02",
+        "Caué": "03",
+        "Lembá": "04",
+        "Lobata": "05",
+        "Mé-Zóchi": "06",
+        # Autonomous Region.
+        "Príncipe": "P",  
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)